owl/EASE-OBJ.owl

```diff
@@ -1876,6 +1876,15 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#GeometricalQuality -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#GeometricalQuality">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-OBJ.owl#Intrinsic"/>
+        <rdfs:comment>An intrinsic quality that is an aspect of the spatial form of an object such as shape, size, sharpness etc</rdfs:comment>
```

Comment:
Other comments end with a dot.

Reply:
I think, when you start with "such as" no etc. is needed at the end of the sentence, or?

Reply:
Maybe better "such as *its* shape, size, or sharpness."?

Reply:
Done.